lib/taurus/external/ordereddict/__init__.py

```diff
@@ -0,0 +1,8 @@
+from __future__ import absolute_import
+
+try:
```

Comment:
I think that we do not need the try-except or the `__future__` import: In Taurus 4 we depend on python 2.7 which provides collections.OrderedDict

So I would simply do `from collections import OrderedDict`

Reply:
Well... actually I would also add a deprecation warning using `taurus.deprecated`

Reply:
And we also should edit the changelog accordingly